test/gtest/common/mem_buffer.h

```diff
@@ -103,8 +105,10 @@ class mem_buffer {
         return m_bar1_free_size;
     }
 
-    mem_buffer(size_t size, ucs_memory_type_t mem_type);
-    mem_buffer(size_t size, ucs_memory_type_t mem_type, uint64_t seed);
+    mem_buffer(size_t size, ucs_memory_type_t mem_type, bool async = false);
+    mem_buffer(size_t size, ucs_memory_type_t mem_type, bool async,
```

Comment:
why not adding it to the end and also default to false?

Reply:
It will not be compiled in this case. Because this will be unclear to the compiler which method to call in case of pasing three arguments.

Reply:
Removed parameter from ctor.